bin/fixtures/config.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+/**
+ * The configuration of SimpleSAMLphp
+ */
+
+// $httpUtils = new \SimpleSAML\Utils\HTTP();
+
+$config = [];
+
+$config['secretsalt'] = 'todo';
+$config['auth.adminpassword'] = 'admin';
+$config['auth.adminsource'] = 'example-userpass'; // this does not appear to exist
```

Comment:
```suggestion
$config['auth.adminsource'] = 'example-userpass'; // this does not appear to exist
```

delete then?

Reply:
this was an old note to self. it's not in the boilerplate config.php but it is a supported param.

```suggestion
$config['auth.adminsource'] = 'example-userpass';
```